src/components/Header.js

```diff
@@ -0,0 +1,15 @@
+// Header.js
+import BaseComponent from "./BaseComponent.js";
+
+export default class Header extends BaseComponent {
+    constructor(page) {
+        super(page, page.locator('header'));
+        // Refine the locator to target the profile button specifically
+        this.profileBtnInHeader = page.locator('button[aria-label="My profile"]');
```

Comment:
just 'profileBtn' would be fine since you will refer to it like `header.profileBtn`